spec/services/post_scraper_spec.rb

```diff
@@ -10,13 +10,13 @@
     url = 'http://wild-pegasus-appeared.dreamwidth.org/403.html?view=flat'
     scraper = PostScraper.new(url)
     expect(scraper.url.sub('view=flat', '')).not_to include('view=flat')
-    expect(scraper.url.gsub("&style=site", "").length).to eq(url.length)
+    expect(scraper.url.length).to eq(url.length)
   end
 
-  it "should add site style to the url" do
-    url = 'http://wild-pegasus-appeared.dreamwidth.org/403.html?view=flat'
+  it "should remove non-site styles from the url" do
+    url = 'http://wild-pegasus-appeared.dreamwidth.org/403.html?style=mine&view=flat'
     scraper = PostScraper.new(url)
-    expect(scraper.url).to include('&style=site')
+    expect(scraper.url).not_to include('&style=mine')
```

Comment:
I'd remove the `&` from the start of this string – `?style=mine`, as in the current string, would falsely pass this test